go.mod

```diff
@@ -10,12 +10,14 @@ require (
 require (
 	github.com/aws/aws-lambda-go v1.28.0 // indirect
 	github.com/gin-contrib/sse v0.1.0 // indirect
+	github.com/gin-contrib/static v0.0.1
```

Comment:
This dependency is not used anymore. I think we should run `go mod tidy` to remove the dependencies from go mod.